client/blocks/app-banner/index.jsx

```diff
@@ -131,9 +132,21 @@ class AppBanner extends Component {
 			}
 		}
 
-		//TODO: update when deferred deep links are available
 		if ( this.isiOS() ) {
-			return 'itms://itunes.apple.com/us/app/wordpress/id335703880?mt=8';
+			switch ( currentSection ) {
+				case EDITOR:
+					return 'https://apps.wordpress.com/get#post';
+				case NOTES:
+					return 'https://apps.wordpress.com/get#notifications';
+				case READER:
+					if ( currentPathFragment.length === 0 ) {
+						return 'https://apps.wordpress.com/get#read';
+					}
+
+					return 'https://apps.wordpress.com/get#' + currentPathFragment;
```

Comment:
Optional, but [temperate literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) are a bit easier to read than string concat.

eg:
```js
return `https://apps.wordpress.com/get#${ currentPathFragment }`;
```

Reply:
Thanks! With my outdated JavaScript knowledge, I didn't know this was a thing :) Definitely looks nicer.